src/lib/convenience/UpdateContextElementResponse.cpp

```diff
@@ -148,7 +148,7 @@ void UpdateContextElementResponse::fill(UpdateContextResponse* ucrsP)
     //
     for (unsigned int aIx = 0; aIx < cerP->contextElement.contextAttributeVector.size(); ++aIx)
     {
-      cerP->contextElement.contextAttributeVector[aIx]->value = "";
+      cerP->contextElement.contextAttributeVector[aIx]->stringValue = "";  // FIXME P10: automacit value -> stringValue change, please review to check if it is safe
```

Comment:
This might be another FIXME where implementation is needed ...
At least setting the valueType to String.
I took a quick look at the function and I can't say I understand why all attrivutes are nulled out ...
Let's look at it together